demo/src/index.tsx

```diff
@@ -20,180 +21,238 @@ import './main.css';
-                    </AccordionItemHeading>
-                    <AccordionItemPanel>{placeholder.panel}</AccordionItemPanel>
-                </AccordionItem>
-            ))}
-        </Accordion>
-
-        <h2 className="u-margin-top">Pre-expanded items</h2>
-
-        <p>
-            If you set <strong>preExpanded</strong>, then you can choose which
-            items are expanded on mount.
-        </p>
-
-        <p>
-            The strings passed to <strong>preExpanded</strong> are directly
-            related to the <strong>uuid</strong> props of{' '}
-            <strong>AccordionItem</strong>.
-        </p>
-
-        <Accordion preExpanded={[placeholders[0].uuid]}>
-            {placeholders.map((placeholder: Placeholder) => (
-                <AccordionItem
-                    key={placeholder.heading}
-                    uuid={placeholder.uuid}
-                >
-                    <AccordionItemHeading>
-                        <AccordionItemButton>
-                            {placeholder.heading}
-                        </AccordionItemButton>
-                    </AccordionItemHeading>
-                    <AccordionItemPanel>{placeholder.panel}</AccordionItemPanel>
-                </AccordionItem>
-            ))}
-        </Accordion>
-
-        <h2 className="u-margin-top">Informative onChange</h2>
-
-        <p>
-            When you use the <strong>onChange</strong> prop, you can get
-            feedback about which items are expanded.
-        </p>
-
-        <p>
-            In this example, we are simply logging the uuids of the expanded
-            items to the console. Have a click around then check your console to
-            see this in action.
-        </p>
-
-        <Accordion
-            // tslint:disable-next-line react-this-binding-issue jsx-no-lambda
-            onChange={(itemUuids: (string | number)[]): void => {
-                // tslint:disable-next-line no-console
-                console.log(itemUuids);
-            }}
-        >
-            {placeholders.map((placeholder: Placeholder) => (
-                <AccordionItem
-                    key={placeholder.heading}
-                    uuid={placeholder.uuid}
+class App extends React.Component<{}, { expanded: UUID[] }> {
+    constructor() {
+        super({});
+        this.state = { expanded: [] };
+        this.handleChange = this.handleChange.bind(this);
+    }
+
+    handleChange(expanded: UUID[]): void {
+        this.setState({ expanded });
+    }
+
+    render(): JSX.Element {
+        return (
+            <>
+                <h1>React Accessible Accordion</h1>
+
+                <p>
+                    React Component for creating an 'Accordion' that adheres to
+                    the{' '}
+                    <a
+                        href="https://www.w3.org/TR/wai-aria-practices-1.1/#accordion"
+                        target="_BLANK"
+                        rel="noreferrer, nofollow"
```

Comment:
Although this is an existing issue in the code I think the `rel` attribute isn't comma separated